### ISSUE TITLE
Chrome Canary 51.0.2699.0 over-width text

### ISSUE THREAD

**Issue by jjt** [jxnblk/fitter-happier-text#25]
OSX Chrome Canary 51.0.2699.0, no errors in console

![screen shot 2016-04-04 at 8 52 04 pm](https://cloud.githubusercontent.com/assets/399469/14270871/5a19d762-faa7-11e5-9bca-a0ba4c55909a.png)

**Comment by kennethormandy** [comment]
I was able to reproduce this, but it’s just the demo site that’s the issue. https://github.com/jxnblk/fitter-happier-text/pull/24 fixes it, so http://rileyjshaw.com/fitter-happier-text/ should already look correct for you on Canary.

**Comment by jjt** [comment]
Yup, that fixes it for me. Will leave this open until #24 gets merged.
